.gitignore

```diff
@@ -1,3 +1,4 @@
 *.tgz
 .DS_Store
-*.swp
\ No newline at end of file
+*.swp
+docker-compose-externaldb.yaml
```

Comment:
this needed?

Reply:
I actually have a compose file for testing the external DB, could include that?  Or too messy/cluttered?

Reply:
given that the repo already has a compose file for docker testing, I would commit the other one too if its useful